libeos-payg/provider.c

```diff
@@ -387,6 +421,48 @@ epg_provider_get_code_format (EpgProvider *self)
   return iface->code_format;
 }
 
+/**
+ * epg_provider_get_code_format_prefix:
+ * @self: a #EpgProvider
+ *
+ * Get the value of #EpgProvider:code-format-prefix
+ *
+ * Returns: the prefix of the code-format expected by this provider
+ * Since: 0.2.0
+ */
+const gchar *
+epg_provider_get_code_format_prefix (EpgProvider *self)
+{
+  g_return_val_if_fail (EPG_IS_PROVIDER (self), NULL);
+
+  EpgProviderInterface *iface = EPG_PROVIDER_GET_IFACE (self);
+
+  g_assert (iface->code_format_prefix != NULL);
```

Comment:
I'd rather have this function return `""` when the field is `NULL`. This would have the added upside that the existing version of eos-payg-nonfree would continue to work (just with these fields empty) rather than crashing.